src/Module/Settings/Display.php

```diff
@@ -153,7 +157,16 @@ protected function post(array $request = [])
 		$this->pConfig->set($uid, 'system', 'preview_mode', $preview_mode);
 		$this->pConfig->set($uid, 'system', 'embed_remote_media', $embed_remote_media);
 		$this->pConfig->set($uid, 'system', 'embed_media', $embed_media);
-
+		if ($widget_timeline_reset == 1) {
```

Comment:
You can simply write `if ($widget_timeline_reset) {`

Reply:
Done.